app/models/jupiter_core/search.rb

```diff
@@ -1,6 +1,6 @@
 class JupiterCore::Search
 
-  # How dumb is this? Seems pretty dumb, but that's the official recommendation I guess:
+  # How unintelligent is this? Seems pretty unintelligent, but that's the official recommendation I guess:
```

Comment:
My personal preference, remove the commentary, something like the following. Feel free to ignore this suggestion. 
```suggestion
  #  official recommendation to return all results in a single page: set to number higher than possible number of results
```